src/gateway/handlers/applicationCommandCreate.ts

```diff
@@ -6,10 +6,7 @@ export const applicationCommandCreate: GatewayEventHandler = async (
   gateway: Gateway,
   d: ApplicationCommandPayload
 ) => {
-  const guild =
-    d.guild_id === undefined
-      ? undefined
-      : await gateway.client.guilds.get(d.guild_id)
+  const guild = await gateway.client.guilds.get(d.guild_id ?? "")

```

Comment:
It will make unnecessary cache request. Imagine user having some custom cache over network.